### PR TITLE
[wpt-sync] Add a preamble with CSSOMString to cssom.idl

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

**Comment by foolip** [comment]
Tested locally to confirm the expected output:
```diff
diff --git a/interfaces/cssom.idl b/interfaces/cssom.idl
index 7d9536d943..a914ac0d78 100644
--- a/interfaces/cssom.idl
+++ b/interfaces/cssom.idl
@@ -1,10 +1,17 @@
+// GENERATED PREAMBLE - DO NOT EDIT
+// This preamble was added by reffy-reports for web-platform-tests.
+// CSSOMString is an implementation-defined type of either DOMString or
+// USVString in CSSOM: https://drafts.csswg.org/cssom/#cssomstring-type
+// For web-platform-tests, use DOMString because USVString has additional
+// requirements in type conversion and could result in spurious failures for
+// implementations that use DOMString.
+typedef DOMString CSSOMString;
+
 // GENERATED CONTENT - DO NOT EDIT
 // Content was automatically extracted by Reffy into reffy-reports
 // (https://github.com/tidoust/reffy-reports)
 // Source: CSS Object Model (CSSOM) (https://drafts.csswg.org/cssom/)
 
-typedef USVString CSSOMString;
-
 [Exposed=Window]
 interface MediaList {
   stringifier attribute [TreatNullAs=EmptyString] CSSOMString mediaText;
@@ -19,7 +26,7 @@ interface StyleSheet {
   readonly attribute CSSOMString type;
   readonly attribute USVString? href;
   readonly attribute (Element or ProcessingInstruction)? ownerNode;
-  readonly attribute StyleSheet? parentStyleSheet;
+  readonly attribute CSSStyleSheet? parentStyleSheet;
   readonly attribute DOMString? title;
   [SameObject, PutForwards=mediaText] readonly attribute MediaList media;
   attribute boolean disabled;
@@ -35,16 +42,16 @@ interface CSSStyleSheet : StyleSheet {
 
 [Exposed=Window]
 interface StyleSheetList {
-  getter StyleSheet? item(unsigned long index);
+  getter CSSStyleSheet? item(unsigned long index);
   readonly attribute unsigned long length;
 };
 
-partial interface Document {
+partial interface mixin DocumentOrShadowRoot {
   [SameObject] readonly attribute StyleSheetList styleSheets;
 };
 
 interface mixin LinkStyle {
-  readonly attribute StyleSheet? sheet;
+  readonly attribute CSSStyleSheet? sheet;
 };
 
 ProcessingInstruction includes LinkStyle;
```

**Comment by foolip** [comment]
@dontcallmedom @tidoust would either of you care to review?

File: wpt-sync/sync.js [hunk 1]
```diff
@@ -14,6 +14,17 @@ If additional changes are needed, please manually create another PR based on thi
...
+const CSSOM_PREAMBLE = `// GENERATED PREAMBLE - DO NOT EDIT
+// This preamble was added by reffy-reports for web-platform-tests.
+// CSSOMString is an implementation-defined type of either DOMString or
+// USVString in CSSOM: https://drafts.csswg.org/cssom/#cssomstring-type
+// For web-platform-tests, use DOMString because USVString has additional
+// requirements in type conversion and could result in spurious failures for
+// implementations that use DOMString.
+typedef DOMString CSSOMString;
+
+`
```
Comment (dontcallmedom) [inline wpt-sync/sync.js]: nit: missing end semi-colon

Comment (foolip) [reply wpt-sync/sync.js]: Added!

**Pull request merged**